numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -730,6 +732,24 @@ VOID_getitem(void *input, void *vap)
         return (PyObject *)ret;
     }
 
+    if (!future_warning_emitted){
+        /* 2017-11-17, 1.14 */
+        if (DEPRECATE_FUTUREWARNING(
+                "the `.item()` method of unstructured void types will return "
+                "an immutable `bytes` object in the near future, the same as "
+                "returned by `bytes(void_obj)`, instead of the mutable "
+                "memoryview or integer array returned in numpy 1.13.")
+                < 0) {
+            return NULL;
+        }
+        future_warning_emitted = 1;
+    }
+    /*
+     * In the future all the code below will be replaced by
+     *     // For unstructured void types like V4, return a bytes object (copy).
```

Comment:
No C++ comments.

Reply:
Technically this is not a C++ comment :)  I'll fix it though.